drivers/SmartThings/matter-button/src/test/test_matter_multi_button.lua

```diff
@@ -14,7 +14,7 @@ local mock_device = test.mock_device.build_test_matter_device(
     manufacturer_info = {vendor_id = 0x0000, product_id = 0x0000},
     endpoints = {
     {
-      endpoint_id = 1,
+      endpoint_id = 10,
```

Comment:
Can you add an endpoint id of 0 to this test device as well. Just want to make sure we get coverage of the common case where there is a root node endpoint without the switch cluster.

Reply:
Added in [a263d34](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/994/commits/a263d34d1dece766e7b46330ca1ab1cc939438ef)